src/App.tsx

```diff
@@ -1,10 +1,22 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState } from 'react';
+
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID } from './utils/todos';
+
+import { Loader } from './components/Loader';
+import { TodoList } from './components/TodoList/TodoList';
+
+import { ErrorNotification } from './components/ErrorNotification';
+
+import { useTodos } from './components/context/TodosContext';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
 
 export const App: React.FC = () => {
+  const [isLoading] = useState(false);
```

Comment:
A state without a setter doesn't make sense.
In the next tasks, you can create a state with an array of todo ids that currently loading, and show loader depending on it